examples/angular_bazel_architect/.bazelrc

```diff
@@ -0,0 +1,86 @@
+# Common Bazel settings for JavaScript/NodeJS workspaces
```

Comment:
To avoid duplication in the repo, change this to 
```
import %workspace%/../../common.bazelrc
<settings specific to this example go HERE>
```

Reply:
Example: https://github.com/bazelbuild/rules_nodejs/blob/master/examples/angular/.bazelrc